test/ceddl.spec.ts

```diff
@@ -24,4 +50,21 @@ describe('mock CEDDL unit tests', () => {
     expect(version).not.be.empty;
   });
 
+
+  it('it should pick data from CEDDL', () => {
```

Comment:
Should we:
A) Reference CEDDL test data in any and all operator's tests
B) Only use mock data defined within the scope of each tests

During prototyping there was feedback that this caused the developer to step out of the test suite to review external test data.  Given we're supporting a specific spec, perhaps we need to simply reference.  This would cut down on the amount of arbitrary test data and provide an artifact for all operator developers.

Reply:
In general, I think it makes sense to test the operators in their own spec files and just test CEDDL itself in here. This should move into operator-pick.spec.ts.

Reply:
Yeah, I think we need a more expressive but lightweight syntax: needs to support dot notation, an array index, and maybe a * wildcard - on properties names and `parent.*`.  Defining the grammar and parser outside would get away from these idiosyncracies.

Reply:
I now see your original question. I think it's fine to test against CEDDL instead of a test-local data structure since it's a standard data layer object that I would guess that most readers will know.